### PR TITLE
Allow to pass connection object

### PR DESCRIPTION
**Pull request by a0s**
Allow to pass connection object in configuration instead of connection string. For example `Rakefile`
```ruby
require File.join(File.dirname(__FILE__), %w{config boot})
require File.join(File.dirname(__FILE__), %w{config initializers sequel})

require 'sequel/rake'
Sequel::Rake.configure do
  set :connection, App.db # App.db was initialized in sequel.rb
  set :migrations, "#{__dir__}/db/migrations"
  set :namespace, 'db'
end
Sequel::Rake.load!
```

### CODE REVIEW

File: lib/sequel/rake/tasks.rake [hunk 1]
```diff
@@ -58,7 +58,11 @@ namespace Sequel::Rake.get(:namespace) do
   end
 
   def connection
-    @connection ||= Sequel.connect(Sequel::Rake.get(:connection))
+    if Sequel::Rake.get(:connection).is_a?(String)
```
Comment (AlexWayfer) [inline lib/sequel/rake/tasks.rake]: [Sequel can receive `Hash`](https://github.com/jeremyevans/sequel/blob/ea79b27/lib/sequel/database/connecting.rb#L42).

Comment (a0s) [reply lib/sequel/rake/tasks.rake]: You're right

File: lib/sequel/rake/tasks.rake [hunk 2]
```diff
@@ -5,7 +5,7 @@ namespace Sequel::Rake.get(:namespace) do
   unless Rake::Task.task_defined?('sequel:environment')
     task :environment
   end
-  
+
```
Comment (AlexWayfer) [inline lib/sequel/rake/tasks.rake]: I'm not sure that this is necessary in this PR.

Comment (a0s) [reply lib/sequel/rake/tasks.rake]: Oop, my master is little different

**Comment by a0s** [comment]
Remove unnecessary changes

**Comment by sandelius** [comment]
Looks good!

**Pull request merged**